### PR TITLE
updateStageInstance, createStageInstance, getStageInstance should use a transformer

### PR DESCRIPTION
**Pull request by lts20050703**
add transformStageInstance transformer
add transformStageInstance transformer to bot.transformers
updateStageInstance, createStageInstance, getStageInstance use transformStageInstance transformer

### CODE REVIEW

File: src/transformers/stageInstance.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+export function transformStageInstance(
+  bot: Bot,
+  payload: SnakeCasedPropertiesDeep<StageInstance>
+): DiscordenoStageInstance {
+  return {
+    id: bot.transformers.snowflake(payload.id),
+    guildId: bot.transformers.snowflake(payload.guild_id),
+    channelId: bot.transformers.snowflake(payload.channel_id),
+    topic: payload.topic,
+    privacyLevel: payload.privacy_level,
```
Comment (Skillz4Killz) [inline src/transformers/stageInstance.ts:15]: delete

File: src/transformers/stageInstance.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    discoverableDisabled: payload.discoverable_disabled,
```
Comment (Skillz4Killz) [inline src/transformers/stageInstance.ts:16]: delete

File: src/transformers/stageInstance.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  /** The id of this Stage instance */
+  id: bigint;
+  /** The guild id of the associated Stage channel */
+  guildId: bigint;
+  /** The id of the associated Stage channel */
+  channelId: bigint;
+  /** The topic of the Stage instance (1-120 characters) */
+  topic: string;
+  /** The privacy level of the Stage instance */
+  privacyLevel: PrivacyLevel;
```
Comment (Skillz4Killz) [inline src/transformers/stageInstance.ts:30]: delete

File: src/transformers/stageInstance.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  /** Whether or not Stage discovery is disabled */
+  discoverableDisabled: boolean;
```
Comment (Skillz4Killz) [inline src/transformers/stageInstance.ts:32]: delete

**Review by Skillz4Killz — CHANGES_REQUESTED** [review]

**Comment by lts20050703** [comment]
@Skillz4Killz nuked privacyLevel and discoverableDisabled

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**